runtime/queries/vhdl/highlights.scm

```diff
@@ -1,338 +1,134 @@
-(comment) @comment
+[
+  "alias" "package" "file" "entity" "architecture" "type" "subtype"
+  "attribute" "to" "downto" "signal" "variable" "record" "array"
+  "others" "process" "component" "shared" "constant" "port" "generic"
+  "generate" "range" "map" "in" "inout" "of" "out" "configuration"
+  "pure" "impure" "is" "begin" "end" "context" "wait" "until" "after"
+  "report" "open" "exit" "assert" "next" "null" "force" "property"
+  "release" "sequence" "transport" "unaffected" "select" "severity"
+  "register" "reject" "postponed" "on" "new" "literal" "linkage"
+  "inertial" "guarded" "group" "disconnect" "bus" "buffer" "body"
+  "all" "block" "access"
+] @keyword
+
+[
+  "function" "procedure"
+] @keyword.function
 
-; Keywords
 [
-  ; vhdl 08
-  "abs"
-  "access"
-  "after"
-  "alias"
-  "all"
-  "and"
-  "architecture"
-  "array"
-  "assert"
-  "attribute"
-  "begin"
-  "block"
-  "body"
-  "buffer"
-  "bus"
-  "case"
-  "component"
-  "configuration"
-  "constant"
-  "disconnect"
-  "downto"
-  "else"
-  "elsif"
-  "end"
-  "entity"
-  "exit"
-  "file"
-  "for"
-  "function"
-  "generic"
-  "group"
-  "guarded"
-  "if"
-  "impure"
-  "in"
-  "inertial"
-  "inout"
-  "is"
-  "label"
-  "library"
-  "linkage"
-  "literal"
-  "loop"
-  "map"
-  "mod"
-  "nand"
-  "new"
-  "next"
-  "nor"
-  "not"
-  "null"
-  "of"
-  "on"
-  "open"
-  "or"
-  "others"
-  "out"
-  "package"
-  "port"
-  "postponed"
-  "procedure"
-  "process"
-  "protected"
-  "pure"
-  "range"
-  "record"
-  "register"
-  "reject"
-  "rem"
-  "report"
   "return"
-  "rol"
-  "ror"
-  "select"
-  "severity"
-  "shared"
-  "signal"
-  "sla"
-  "sll"
-  "sra"
-  "srl"
-  "subtype"
-  "then"
-  "to"
-  "transport"
-  "type"
-  "unaffected"
-  "units"
-  "until"
-  "use"
-  "variable"
-  "wait"
-  "when"
-  "while"
-  "with"
-  "xnor"
-  "xor"
-  ; vhdl 08
-  "context"
-  "force"
-  "property"
-  "release"
-  "sequence"
-] @keyword
+] @keyword.control.return
 
 [
-  ; vhdl 02
-  "boolean"
-  "bit"
-  "bit_vector"
-  ;"character"
-  ;"severity_level"
-  ;"integer"
-  ;"real"
-  ;"time"
-  ;"natural"
-  ;"positive"
-  "string"
-  ;"line"
-  ;"text"
-  ;"side"
-  ;"unsigned"
-  ;"signed"
-  ;"delay_length"
-  ;"file_open_kind"
-  ;"file_open_status"
-  ;"std_logic"
-  ;"std_logic_vector"
-  ;"std_ulogic"
-  ;"std_ulogic_vector"
-  ; vhdl 08
-  ;"boolean_vector"
-  ;"integer_vector"
-  ;"real_vector"
-  ;"time_vector"
-  ; math types
-  ;"complex"
-  ;"complex_polar"
-  ;"positive_real"
-  ;"principal_value"
-] @type.builtin
+  "for" "loop" "while"
+] @keyword.control.repeat
+
+[ 
+  "if" "elsif" "else" "case" "then" "when"
+] @keyword.control.conditional
+
+[ 
+  "library" "use"
+] @keyword.control.import
+
+(comment) @comment @spell
```

Comment:
```suggestion
(comment) @comment
```

`@spell` is nvim specific, they mix their highlight and spell checking queries. For Helit the scope can be removed

Reply:
Done